build/xtask/src/config.rs

```diff
@@ -780,7 +780,7 @@ impl BuildConfig<'_> {
 
         cmd.arg(
             "-Zallow-features=asm_sym,asm_const,named-profiles,naked_functions,\
-cmse_nonsecure_entry,array_methods,backtrace,proc_macro_span,proc_macro_span_shrink,const_trait_impl,\
+cmse_nonsecure_entry,array_methods,backtrace,proc_macro_span,proc_macro_span_shrink,provide_any,const_trait_impl,\
```

Comment:
Is this a side-effect of updating all dependencies in Cargo.lock? I guess @FawazTirmizi didn't think it was a good idea to update all dependencies?

Reply:
Yes - this was a consequence of dependencies changing their unstable feature requirements without bumping their version number to suggest incompatibility. I'm working on a 'thinner' version of this change where we don't have to bump everything.